src/main/resources/db/changelog/changes/2024.06.01-add-task-comment-entity.yaml

```
@@ -0,0 +1,51 @@
+databaseChangeLog:
+  - changeSet:
+      id: 0
+      author: VladisStep
+      comment: add task comment table
+      changes:
+        - createTable:
+            tableName: task_comment
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: create_date
+                  type: timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  name: creator_id
+                  type: bigint
+              - column:
+                  name: task_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: text
+                  type: varchar(1000)
```

```suggestion
                  type: text
```
Мб на текст поменять? Там же длинный коммент в теории может быть, больше 1000